File: C/Estruturas Condicionais/Exercício 4/menor_de_tres.c
#include <stdio.h>

int main() {
    int primeiroValor, segundoValor, terceiroValor;

    printf("Primeiro valor: ");
    scanf("%d", &primeiroValor);

    printf("Segundo valor: ");
    scanf("%d", &segundoValor);

    printf("Terceiro valor: ");
    scanf("%d", &terceiroValor);

    if (primeiroValor < segundoValor && primeiroValor < terceiroValor) {
        printf("MENOR = %d", primeiroValor);
    } else if (segundoValor < primeiroValor && segundoValor < terceiroValor) {
        printf("MENOR = %d", segundoValor);
    } else {
        printf("MENOR = %d", terceiroValor);
    }
    return 0;
}

File: Python/Estruturas Condicionais/menor_de_tres.py
primeiroValor: int; segundoValor: int; terceiroValor: int

primeiroValor = int(input("Primeiro Valor: "))
segundoValor = int(input("Segundo Valor: "))
terceiroValor = int(input("Terceiro Valor: "))

if primeiroValor < segundoValor and primeiroValor < terceiroValor:
    print(f"MENOR = {primeiroValor}")
elif segundoValor < primeiroValor and segundoValor < terceiroValor:
    print(f"MENOR = {segundoValor}")
else:
    print(f"MENOR = {terceiroValor}")